website/aws.erb

```diff
@@ -1823,6 +1823,9 @@
                                 <li>
                                     <a href="/docs/providers/aws/r/guardduty_organization_configuration.html">aws_guardduty_organization_configuration</a>
                                 </li>
+                                <li>
```

Comment:
This shared file that always caused merge conflicts is now removed, so changes here should be removed. 😄